contracts/0.8.9/lib/PositiveTokenRebaseLimiter.sol

```diff
@@ -93,22 +135,22 @@ library PositiveTokenRebaseLimiter {
         pure
         returns (uint256 consumedEther)
     {
-        if (_limiterState.rebaseLimit == UNLIMITED_REBASE) {
-            return _etherAmount;
-        }
+        if (_limiterState.rebaseLimit == UNLIMITED_REBASE) return _etherAmount;
 
-        uint256 remainingRebase = _limiterState.rebaseLimit - _limiterState.accumulatedRebase;
-        uint256 remainingEther = (remainingRebase * _limiterState.totalPooledEther) / LIMITER_PRECISION_BASE;
+        uint256 prevPooledEther = _limiterState.postTotalPooledEther;
+        _limiterState.postTotalPooledEther += _etherAmount;
 
-        consumedEther = Math256.min(remainingEther, _etherAmount);
+        uint256 rebaseEtherLimit =
```

Comment:
`+  _limiterState.preTotalPooledEther` and rename to maxTotalPooledEther?

Reply:
:+1: